### PR TITLE
Hide zones with insufficient data from the map

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Issue

We have some zones that are always showing the same carbon intensity for every single hour of the year. This means we are showing potentially quite wrong data (Thailand as example has an average of 550g co2/kWh for 2023 according to [Our World in Data](https://ourworldindata.org/grapher/carbon-intensity-electricity?tab=chart&time=2017..latest&country=~THA) vs our 409g CO2/kWh).

## Description

This PR hides certain zones from the map to avoid showing incorrect colors and numbers on the map.
I decided to reuse the `aggregates_displayed` property in zone configurations to avoid introducing more new ways of doing things. To make changes minimal I added a "none" entry for the zones, but an alternative implementation (which requires a bit more work) would be to return an empty array instead (`aggregates_displayed: []`).

**Zones hidden:**
- All subzones in Mexico
- Thailand
- Kuwait
- Bahrain

### Preview

Screenshot 2024-06-18 at 22 48 17

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/src/locales/en.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
     "helpUs": "Help us identify the problem by taking a look at the <a href=\"{{link}}\" target=\"_blank\">runtime logs</a> or contact the data provider.",
     "noParserInfo": "We do not yet have any information about this area.<br/> Want to help fix this? You can contribute by <a href=\"{{link}}\" target=\"_blank\">adding area data</a>.",
     "aggregationDisabled": "Data is not available for the selected time period due to missing granular breakdown. Try a broader time period.",
+    "fullyDisabled": "The data we have available for this zone is insufficient and therefore not shown. If you are aware of data sources for this area, please <a href=\"{{link}}\" target=\"_blank\">let us know</a>.",
```
Comment (VIKTORVAV99) [inline web/src/locales/en.json:91]: Just a small nitpick here, it wound be nice to use single quotes in the links so we can avoid the escape chars. But I see it's the same in all the other ones I guess it's fine.

File: web/src/features/panels/zone/util.ts [hunk 2]
```diff
@@ -44,15 +45,16 @@ export const getZoneDataStatus = (
   // If there is no config for the zone, we assume we do not have any data
   const zoneConfig = config.zones[zoneId];
   if (!zoneConfig) {
-    console.log(zoneConfig);
-
```
Comment (VIKTORVAV99) [inline web/src/features/panels/zone/util.ts:48]: you can potentially hide this in a
`if (import.meta.env.DEV){}` clause instead if you want it for debug builds.

**Review by VIKTORVAV99 — COMMENTED** [review]
Does this require the backend to be updated/use this data as well to function or should it work in just the app?

It doesn't seem to work in the preview links.

File: web/src/locales/en.json (hunk 1 above)
Comment (madsnedergaard) [reply web/src/locales/en.json:91]: Yeah, I just copy-pasted - but happy to make an issue for generally fixing this problem afterwards :)

File: web/src/features/panels/zone/util.ts (hunk 2 above)
Comment (madsnedergaard) [reply web/src/features/panels/zone/util.ts:48]: It was an old left-over console log that seemed like a mistake, so I just removed it :)

**Comment by madsnedergaard** [comment]
> Does this require the backend to be updated/use this data as well to function or should it work in just the app?

Backend already works with no changes with this approach :)

File: web/src/locales/en.json (hunk 1 above)
Comment (VIKTORVAV99) [reply web/src/locales/en.json:91]: Sounds good, something to do next slack week!

**Review by VIKTORVAV99 — APPROVED** [review]
LGMT! 🎉

File: web/src/locales/en.json (hunk 1 above)
Comment (madsnedergaard) [reply web/src/locales/en.json:91]: Captured in Linear here: https://linear.app/electricitymaps/issue/AVO-336/remove-escaped-quotes-from-translation-files

**Pull request merged**